### DISCUSSION TITLE
一篇流水账 | /posts/flying-water/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#614] [Announcements]
https://yuanfan.rbind.io/posts/flying-water/

**Comment by HushWay** [comment]
前几段风的描写与想象唤起了读散文的感觉，小时候觉得散文很无聊，远没有小说来劲。年纪大了，读来才发觉别有风味，没过瘾，忍不住找来朱自清的《匆匆》读了几遍。

你说梨子市，直觉是金川，因为半个月前看到一张金川万里梨花图，绵延千里，着实壮观

最后，原来好多人被夜晚的情绪摧残。最近看到一句话，焦虑的反义词不是平静，而是具体。我琢磨情绪很大程度上真是未知的引起，期待或焦虑。所以，我想如果要克服一个事情对自己的影响的话，还是要面对，想象它具体的发生，想象具体的对策，采取具体的对策，还是没办法那就接受，没什么大不了的。

**Reply by earfanfan** [reply]
哇，谢谢你，你好擅长发现我的闪光点啊，哈哈，我的博客里大多都是絮絮叨叨记录一些生活的，好久好久都没人来夸夸我了，我发现有读者夸夸我，还挺让人高兴的呢。

金川我此前没听过，倒是知道存在银川和铜川……上网搜了搜原来金川在四川，我在湖北，由于文中涉及了一些政府相关的，所以写的时候随便写了个别名。

最后，关于情绪，我认同要面对，不过期待和焦虑这两种情绪本来也是自己的一部分，我最近想的是也许并不是要克服情绪对自己的影响，因为如果是要克服的话就好像自己在面对撒很坏的影响，实际上情绪波动也是身体与心灵碰到一些事的反应。我小时候受到所处环境的影响对一些情绪波动很抵触，比如书本上写要“戒骄戒躁”，这像是把骄傲、烦躁、暴躁等情绪定义为一种负面的东西，既然定义了负面那自然也是定义了正面的，可我觉得不该用那么刻板的标准去定义自己的反应，而是要更多地认识自己，比如超级伤心的时候暴风哭泣，那就是真地很伤心很难受啊哭出来心里才能舒服点，又比如焦虑或者内耗，换个角度看有另一种说法“生于忧患死于安乐”，内耗也确实会让人觉得难受，可是（仅对我而言）不经历内耗也不会看清楚自己到底怎么了。

我想就把情绪当做情绪吧，先不用什么标准去定义情绪是正面还是负面，而是先识别清楚是哪一种情绪，然后根据找到的情绪诞生的源头再看咋面对。当然，一些很浅很轻的情绪不用这样，因为一下子就过去了，这就不用分析来分析去地搞得那么沉重，但是那些重一些的情绪，类似于在情绪漩涡里打转的或者总被某种情绪浪潮拍得疼的，就需要先面对情绪造成的影响，如果感到伤心就是受伤了，受伤了就会有伤口，有伤口就需要治愈，治愈就需要时间，那就需要飘然等待。好在现代人生活都很丰富，等待的时候也可以做很多事情。总而言之，面对具体的事情就想具体的解决办法，但是不要戒掉自己的情绪。

**Reply by HushWay** [reply]
你们人类真奇妙，本三体人还要多体验才能理解
